.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+	"printWidth": 120,
+	"trailingComma": "all",
+	"singleQuote": true,
+	"useTabs": true
```

Comment:
Tabs is the new standard. This code base will correct to tabs over time with this change.